projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/blocks/feeder.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "饲养板",
+    "icon": "productivebees:feeder",
+    "category": "productivebees:blocks",
+    "pages": [
+        {
+            "type": "patchouli:spotlight",
+            "item": "productivebees:feeder",
+            "text": "可用作喂养蜜蜂的集散点。包含 3 个物品栏。当木场蜜蜂或石料蜜蜂在此采蜜时，它会用饲养板中的随机物品进行生产。"
+        },
+        {
+            "type": "patchouli:text",
+            "text": "喂食板可以倒上蜂蜜，如此就为蜜蜂提供了一个泡泡脚、从一天的漫长工作中恢复过来的好地方，即给予它们再生效果。"
+        },
+        {
+            "type": "patchouli:text",
+            "text": "由于$(l:items/upgrades#simulator)升级：模拟$(/l)只有 1 个花块位置，饲养板可以与它达成很好的配合，让您有最多 3 只不同的蜜蜂在蜂巢中工作。"
```

这里的flowerblock是专用名称吗

---

应该是说给蜂箱装了模拟升级后，蜂箱里的蜜蜂就只探测蜂箱面前这一格方块正下方是否是饲养板，这一格方块就叫flowerblock。可能是这样吧😥

---

可能还需要再测试
没关系先改别的位置，改完叫我我再看一遍

---

不改了，确保各个文件统一即可
```suggestion
            "text": "由于$(l:items/upgrades#simulator)升级：模拟$(/l)只有 1 个花块位置，饲养板可以与它达成很好的配合，让你有最多 3 只不同的蜜蜂在蜂巢中工作。"
```
求求了别再用您了

---

WTF这个您替换漏了